Makefile

```diff
@@ -77,19 +79,25 @@ $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
 		fi
 	fi
 
-	if [ -f "$(NON_UP_DIR)/series.patch" ]; then
-		echo "Patch the series file"
-		cat $(NON_UP_DIR)/series.patch
-		pushd patch
-		# clear any unstaged changes
-		git stash -- series
-		git apply $(NON_UP_DIR)/series.patch
-		popd
-
-		if [ -d "$(NON_UP_DIR)/patches" ]; then
-			echo "Copy the non upstream patches"
-			cp $(NON_UP_DIR)/patches/*.patch patch/
-		fi
+	pushd patch
+	if [ -f "$(NON_UP_DIR)/$(SERIES).patch" ]; then
+		echo "Patch the $(SERIES) file"
+		cat $(NON_UP_DIR)/$(SERIES).patch
+		git stash -- $(SERIES)
+		git apply $(NON_UP_DIR)/$(SERIES).patch
+	fi
+
+	if [ -f "$(NON_UP_DIR)/$(KCFG_INCL).patch" ]; then
+		echo "Patch the $(KCFG_INCL) file"
+		cat $(NON_UP_DIR)/$(KCFG_INCL).patch
+		git stash -- $(KCFG_INCL)
+		git apply $(NON_UP_DIR)/$(KCFG_INCL).patch
+	fi
```

Comment:
Why are two separate patches needed here? A single patch should be able to handle everything.

Reply:
Thanks for pointing out, will update

Reply:
Handled